wallet-new/test/Cardano/Wallet/WalletLayer/QuickCheck.hs

```diff
@@ -132,3 +138,27 @@ instance Arbitrary DeleteWalletError where
 
 instance Arbitrary UpdateWalletError where
     arbitrary = oneof [ UpdateWalletWalletIdDecodingFailed <$> arbitrary ]
+
+
+instance Arbitrary GetUtxosError where
+    arbitrary = oneof [ GetWalletUtxosWalletIdDecodingFailed <$> arbitrary
+                      , GetUtxosErrorFromGetAccountsError <$> arbitrary
+                      ]
+
+genTxIn :: Gen TxIn
+genTxIn = oneof
+    [ TxInUtxo <$> arbitrary <*> arbitrary
+    , TxInUnknown <$> choose (1, 255) <*> scale (min 150) arbitrary
+    ]
```

Comment:
Why have this defined here as a separated function :thinking: ?

Reply:
was reused in one of my earlier versions, now only in one place -> fixed